docs/ecosystem/oracles/pragma.md

```diff
@@ -1,3 +1,147 @@
 # Pragma Oracle
 
-Under construction 🏗️
+Pragma is the decentralized, transparent and composable oracle network, leveraging state-of-the-art zero-knowledge cryptography. We partner with the biggest market makers and the most liquid exchanges who sign and timestamp their own high quality, robust data and send it directly on-chain.
+
+Our data feeds are live on 🥕 Kakarot, where they leverage the power of the ZK-EVM. Our code has been audited by leading security researchers at Nethermind and we have more audits coming up soon.
+
+We are currently deployed on the following addresses:
+
+| Network | Cairo Address                                                     | PragmaCaller Solidity Contract                                                                                                   |
```

Comment:
I would suggest to put first the PragmaCaller Solidity Contract, since most readers of the docs are Solidity devs

And Cairo Address isn't so clear for non-experts, I'd suggest calling it `Native Starknet Address (ignore if you're an EVM developer)`

Reply:
Should be good!